bitnami/aspnet-core/values.yaml

```diff
@@ -80,6 +80,9 @@ image:
   ##
   pullSecrets: []
   debug: false
+## @param useContainerCommand Use container command instead of override it (useful when using custom images)
+##
+useContainerCommand: false
 ## @param command Override default container command (useful when using custom images)
```

Comment:
We should add some comments about the new logic:

```suggestion
## @param command [object,nullable] Override custom pod command (useful when using custom images)
## If empty, it will default to "dotnet". Set to null to use the default container command 
```